src/main/kotlin/com/kotlin/study/dongambackend/domain/notice/service/NoticeService.kt

```diff
@@ -0,0 +1,51 @@
+package com.kotlin.study.dongambackend.domain.notice.service
+
+import com.kotlin.study.dongambackend.common.exception.BaseException
+import com.kotlin.study.dongambackend.common.type.ResponseStatusType
+import com.kotlin.study.dongambackend.domain.notice.dto.request.NoticeCreateRequest
+import com.kotlin.study.dongambackend.domain.notice.entity.Notice
+import com.kotlin.study.dongambackend.domain.notice.repository.NoticeRepository
+import com.kotlin.study.dongambackend.domain.notice.dto.request.NoticeUpdateRequest
+import com.kotlin.study.dongambackend.domain.notice.dto.response.NoticeCategoryFreeResponse
+import com.kotlin.study.dongambackend.domain.notice.repository.NoticeQueryDslRepository
+import lombok.extern.slf4j.Slf4j
+import org.springframework.data.domain.Pageable
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.io.IOException
+
+@Service
+@Slf4j
+class NoticeService(private val noticeRepository: NoticeRepository, private val noticeQueryDslRepository: NoticeQueryDslRepository,) {
+
+    fun createNotice(noticeCreateRequest: NoticeCreateRequest): Long? {
+        val notice = Notice(noticeCreateRequest.content)
+        noticeRepository.save(notice)
```

Comment:
```suggestion
        return noticeRepository.save(notice).id
```

[선택] 이렇게 하시면 코드 좀 짧게 줄일 수 있을 것 같아요~!